.dockerignore

```diff
@@ -2,3 +2,4 @@
 **/npm-debug.log
 build
 dist
+camino-wallet-sdk
```

Comment:
maybe I'm wrong, but what I understand is that we need camino-wallet-sdk for building camino wallet. Why do you remove copying it inside the docker. in the docker file you copy the sourcecode and then run yarn build. won't this affect the build inside the docker ?